.github/workflows/chapter-4-contracts-workflow.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         with:
           dotnet-version: 8.0.x
       - name: Add Evolutionary Architecture Nuget Source
-        uses: evolutionary-architecture/evolutionary-architecture-by-example/.github@main
+        uses: evolutionary-architecture/evolutionary-architecture-by-example/.github@feature/add_signature_value_object
```

Comment:
This is a temporary change. The current version on the main branch isn’t working, so I will merge this one for now. After merging, I will create a dedicated merge request to revert it back to the main version.